v2/codegen/infragen/secretsgen/secretsgen.go

```diff
@@ -27,12 +29,21 @@ func Gen(gen *codegen.Generator, pkg *pkginfo.Package, secrets []*secrets.Secret
 			addedImport[secret.File] = true
 		}
 
+		svcName := strconv.Quote(
+			option.
+				Map(
+					svc,
+					func(svc *app.Service) string { return svc.Name },
+				).
+				GetOrElse(""),
+		)
```

Comment:
Can we make this like two lines? If we move the closure to a separate variable the rest should fit in one line